modules/web/src/app/dynamic/enterprise/metering/schedule-config/edit-dialog/template.html

```diff
@@ -58,6 +58,26 @@
              target="_blank">here. </a> Please note that specifying seconds is not supported.
         </mat-hint>
       </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Report Types</mat-label>
+        <mat-select [formControlName]="controls.Types"
+                    multiple
+                    panelClass="km-multiple-values-dropdown"
+                    disableOptionCentering
+                    required>
```

Comment:
i think you need to fetch the latest main branch in order to add `kmValueChangedIndicator` directive?